src/components/Proposal/Staking/BoostAmount.tsx

```diff
@@ -26,32 +25,34 @@ export default class BoostAmount extends React.Component<IProps, null> {
       [css.wrapper]: true,
       [css.detailView]: detailView,
     });
+    const nothing = <span className={css.boostedAmount}><b>&nbsp;</b></span>;
 
     return (
       <div className={wrapperClass}>
         {
-          proposal.stage === IProposalStage.Queued && !expired && proposal.upstakeNeededToPreBoost.gte(new BN(0)) ?
+          !expired ? ((proposal.stage === IProposalStage.Queued && proposal.upstakeNeededToPreBoost.gten(0)) ?
```

Comment:
I dont think this is right, cant you still boost/unboost a pre-boosted proposal that is expired? or do we want to turn that off

Reply:
I was misinterpreting what "expired" meant (forgetting that it often means expired only within the context of the current phase, but not necessarily that the proposal is dead, and not necessarily that the expired state is even represented yet in the blockchain).

So good catch.  I have reverted the change.